src/main/java/mate/project/model/User.java

```diff
@@ -0,0 +1,32 @@
+package mate.project.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false, unique = true)
+    private String email;
+    @Column(nullable = false)
+    private String password;
```

Comment:
The 'password' field should be encrypted and not stored as plain text for security reasons. It's best to use encryption methods or hashing algorithms to store passwords.

Reply:
I am not too sure how to implment this yet, I believe in further lessons there is a video and theory explaining this, I will make sure to change it then